src/routes/common/utils/utils.ts

```diff
@@ -1,3 +1,17 @@
 export function isHex(value: unknown): boolean {
   return typeof value === 'string' && value.startsWith('0x');
 }
+
+export const shortenAddress = (address: string, length = 4): string => {
+  if (!address) {
+    return '';
+  }
+
+  const visibleCharactersLength = length + 2 * 2;
```

Comment:
What's the reason behind the `+2` and the `*2`?

Reply:
The result address is 0x-prefixed so we need to count that in and multiply length by 2 because thats the number of characters visible from the start and end of the address i.e. `0x${length characters}...{length characters}`. It should actually be `length * 2 + 2` to work. Will fix it and also write some tests.

Reply:
I'd also suggest adding docs to this function so that it's more clear what it does 🙂